app/views/kaui/accounts/index.html.erb

```diff
@@ -42,12 +43,26 @@ $(document).ready(function() {
     "pageLength": <%= @limit %>,
     "displayStart": <%= @offset %>,
     <% unless @ordering.blank? %>
-        "order": [[ 0, "<%= @ordering %>" ]],
+        "order": [[ 1, "<%= @ordering %>" ]],
```

Comment:
Why this change? It's not consistent with other screens anymore.

Reply:
The change was because the first column was not the name anymore but a tag if the account is a child, and it will cause the UI to show an arrow when the first column is not orderable. I could restore it as it was, and mark the account as a child with an icon (http://fontawesome.io/icon/child/) after the name.

Reply:
Oh, you are right. My bad, I misunderstood the API. I thought this changed desc to asc or vice versa.

:shipit: